modules/smiley.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+'''
+smiley.py - Smiley Module
+Copyright 2014 Sujeet Akula (sujeet@freeboson.org)
+Licensed under the Eiffel Forum License 2.
+
+More info:
+ * jenni-misc: https://github.com/freeboson/jenni-misc/
+ * jenni: https://github.com/myano/jenni/
+ * Phenny: http://inamidst.com/phenny/
+'''
+
+import io, random, os
+
+def smiley(jenni, input):
+    '''.smiley -- print a random smiley'''
+
+    with io.open(os.getcwd() + '/modules/smileys.txt', 'r', encoding='utf-16-le') as f:
```

Comment:
smileys.txt has to be in modules/ so that it can be fetched when cloning jenni... the alternative would be to fetch it from a server on first run as is done for twss.py